src/distribution.jl

```diff
@@ -117,8 +117,9 @@ end
 
 Shortcut for `mean(map(f, dist))`.
 """
-function Statistics.mean(f, dist::FixedAtomsProbabilityDistribution)
-    return mean(map(f, dist))
+function Statistics.mean(f, dist::FixedAtomsProbabilityDistribution; kwargs...)
+    fk = FixKwargs(f, kwargs)
+    return mean(map(fk, dist))
 end
```

Comment:
I don't think it's a good idea to introduce a new API for an existing function. No implementation of `Statistics.mean` accepts kwargs, there's no reason ours should:

```julia
julia> using Statistics

julia> methods(mean)
# 6 methods for generic function "mean" from Statistics:
 [1] mean(f::Number, itr::Number)
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:106
 [2] mean(f, A::AbstractArray; dims)
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:104
 [3] mean(f, itr)
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:61
 [4] mean(r::AbstractRange{<:Real})
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:195
 [5] mean(A::AbstractArray; dims)
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:174
 [6] mean(itr)
     @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Statistics/src/Statistics.jl:44
```

Reply:
agree with you, thought I reverted it the other day, now it's done